.buildkite/pipeline.linux.yml

```diff
@@ -0,0 +1,158 @@
+steps:
+  - label: "build-gnu-x86"
+    commands:
+     - cargo build --release
+    retry:
+      automatic: false
+    agents:
+      platform: x86_64.metal
+      os: linux
+    plugins:
+      - docker#v3.0.1:
+          image: "fandree/rust-vmm-dev"
+          always-pull: true
+
+  steps:
```

Comment:
The pipeline upload fails with:
```
2019-05-30 11:58:11 FATAL  Pipeline parsing of "pipeline.linux.yml" failed (Failed to parse pipeline.linux.yml: line 14: did not find expected '-' indicator)
--
```

There should be only on "steps" per yaml file. The additional ones need to be removed. Same for the windows pipeline. This might be a leftover from the previous design.

Reply:
Whops, sorry about that. I've updated the commit.